src/utility/commonKeys.js

```diff
@@ -0,0 +1,16 @@
+// keep consistent with codemirror:
+// https://github.com/codemirror/CodeMirror/blob/ed8dfeb5e2/src/util/browser.js#L17
+const isMacOs = /Mac OS X 1\d\D([8-9]|\d\d)\D/.test(navigator.userAgent);
```

Comment:
Rather than sniffing the user agent, let's just use [the `platform` API](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/platform):

```suggestion
const isMacOs = window.navigator.platform === 'MacIntel';
```

I don't think we need to worry about 'Mac68K' and 'MacPPC' which are other valid options for Mac platforms.

Reply:
Updated